docs/DeveloperGuide.adoc

```diff
@@ -177,7 +177,7 @@ Step 2. The user executes `delete 5` command to delete the 5th person in the add
 
 image::UndoRedoState1.png[]
 
-Step 3. The user executes `add n/David ...` to add a new person. The `add` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
+Step 3. The user executes `insert n/David ...` to add a new person. The `insert` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
```

Comment:
> Step 3. The user executes `insert n/David ...` to add a new person.
Perhaps this sentence, can change from to "to add a new person." to "to add a new delivery orders."

Reply:
this includes all the to add new person or adding person should be changed to inserting order delivery details or things along that line.

> Step 4. The user now decides that adding the person was a mistake, and decides to undo that action by executing the `undo` command. The `undo` command will call `Model#undoAddressBook()`, which will shift the `currentStatePointer` once to the left, pointing it to the previous address book state, and restores the address book to that state.

If possible can change whatever you can spot if not when we do a review standardising in later stage we can keep a look out on such details